scripts/setup_msi.py

```diff
@@ -36,6 +39,7 @@
         ROOT_DIR / "assets" / "pixel-pycb.ico",
         ROOT_DIR / "pyclashbot" / "detection" / "reference_images",
         ROOT_DIR / "pyclashbot" / "__version__",
+        (TTKBOOTSTRAP_PATH, Path("lib") / "ttkbootstrap"),
```

Comment:
can we use the `packages` build exe option here?
`"packages": ["tkinter", "ttkbootstrap"]`

this seems a bit cleaner than importing and directly including the files imo